vnext/ReactUWP/Utils/ValueUtils.cpp

```diff
@@ -49,13 +50,51 @@ REACTWINDOWS_API_(winrt::Color) ColorFrom(const folly::dynamic& d)
 
 REACTWINDOWS_API_(winrt::SolidColorBrush) SolidColorBrushFrom(const folly::dynamic& d)
 {
-  thread_local static std::map<winrt::Color, winrt::weak_ref<winrt::SolidColorBrush>, ColorComp> solidColorBrushCache;
+  if (d.isObject())
+  {
+    winrt::hstring resourceName{ winrt::to_hstring(d.find("windowsbrush")->second.asString()) };
+
+    thread_local static std::map<winrt::hstring, winrt::weak_ref<winrt::SolidColorBrush>> accentColorMap = {
+      {L"SystemAccentColor",            { nullptr }},
+      {L"SystemAccentColorLight1",      { nullptr }},
+      {L"SystemAccentColorLight2",      { nullptr }},
+      {L"SystemAccentColorLight3",      { nullptr }},
+      {L"SystemAccentColorDark1",       { nullptr }},
+      {L"SystemAccentColorDark2",       { nullptr }},
+      {L"SystemAccentColorDark3",       { nullptr }},
+      {L"SystemListAccentLowColor",     { nullptr }},
+      {L"SystemListAccentMediumColor",  { nullptr }},
+      {L"SystemListAccentHighColor",    { nullptr }}
+    };
+
+    if (accentColorMap.find(resourceName) != accentColorMap.end())
+    {
+      if (auto brush = accentColorMap.at(resourceName).get())
+      {
+        return brush;
+      }
+
+      winrt::hstring xamlString =
+        L"<ResourceDictionary xmlns = 'http://schemas.microsoft.com/winfx/2006/xaml/presentation' xmlns:x = 'http://schemas.microsoft.com/winfx/2006/xaml'>"
+        L"  <SolidColorBrush x:Key='" + resourceName + L"' Color='{ThemeResource " + resourceName + "}' />"
+        L"</ResourceDictionary>";
+
+      auto dictionary{ winrt::unbox_value<winrt::ResourceDictionary>(winrt::Markup::XamlReader::Load(xamlString)) };
+      auto brush{ winrt::unbox_value<winrt::SolidColorBrush>(dictionary.Lookup(winrt::box_value(resourceName))) };
+      accentColorMap[resourceName] = winrt::make_weak(brush);
+      return brush;
+    }
+
+    winrt::IInspectable resource{ winrt::Application::Current().Resources().Lookup(winrt::box_value(resourceName)) };
+    return winrt::unbox_value<winrt::SolidColorBrush>(resource);
```

Comment:
>return winrt::unbox_value<winrt::SolidColorBrush>(resource); [](start = 4, length = 60)

If the app supplies a name that isn't recognized, will we crash?  Should we fallback to a transparent brush instead (or some other fallback)?